README.md

```diff
@@ -17,13 +17,13 @@ The consolidated OCDS for PPPs extension can be declared in [OCDS package metada
 }
 ```
 
-## About PPP extensions 
+## About PPP extensions
 
 OCDS for PPPs is constructed from [a number of different modular extensions to OCDS](http://standard.open-contracting.org/profiles/ppp/latest/en/extensions/), most of which can be used independently from the profile.
 
-This repository contains one additional extensions that forms part of the OCDS for PPPs profile. This extension introduces a number of fields and building blocks that are specific to PPP disclosure against the World Bank Framework, as well as a set of modifications that only apply in an OCDS Profile (e.g. removing standard fields that are not required in the profile). 
+This repository contains one additional extensions that forms part of the OCDS for PPPs profile. This extension introduces a number of fields and building blocks that are specific to PPP disclosure against the World Bank Framework, as well as a set of modifications that only apply in an OCDS Profile (e.g. changing closed codelists).
```

Comment:
If https://github.com/open-contracting-extensions/public-private-partnerships/issues/237 is closed, then we can remove the last clause entirely.

```suggestion
This repository contains one additional extension that forms part of the OCDS for PPPs profile. This extension introduces a number of fields and building blocks that are specific to PPP disclosure against the World Bank Framework, as well as a set of modifications that only apply in an OCDS Profile (e.g. changing closed codelists).
```